tests/unit/Injector.ts

```diff
@@ -19,13 +25,30 @@ registerSuite({
 		const testProperties = {
 			qux: 'baz'
 		};
+
+		class TestWidget extends WidgetBase<any> {
+			render() {
+				return v('div', { testFunction: this.properties.testFunction });
+			}
+		}
+
+		function testFunction(this: any) {
+			assert.strictEqual(this, bind);
+		}
+
 		const testChildren: DNode[] = [ 'child' ];
-		const render = (): DNode => { return 'Called Render'; };
+		const render = (): DNode => { return v('div', {}, [
```

Comment:
This test will need to change once bind is not supported in as a property to `v`